appveyor.yml

```diff
@@ -45,13 +45,23 @@ install:
 - cmd : set PNG_LIBRARY_RELEASE=C:\projects\llassetgen\libpng\build\native\lib_release\libpng16.lib
 - cmd : set PNG_LIBRARY_DEBUG=C:\projects\llassetgen\libpng\build\native\lib_debug\libpng16d.lib
 
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" ( set Qt5Core_DIR=C:\Qt\5.9.2\msvc2015_64\lib\cmake\Qt5Core)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" ( set Qt5Gui_DIR=C:\Qt\5.9.2\msvc2015_64\lib\cmake\Qt5Gui)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" ( set Qt5OpenGL_DIR=C:\Qt\5.9.2\msvc2015_64\lib\cmake\Qt5OpenGL)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2015" ( set Qt5Widgets_DIR=C:\Qt\5.9.2\msvc2015_64\lib\cmake\Qt5Widgets)
+
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( set Qt5Core_DIR=C:\Qt\5.9.2\msvc2017_64\lib\cmake\Qt5Core)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( set Qt5Gui_DIR=C:\Qt\5.9.2\msvc2017_64\lib\cmake\Qt5Gui)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( set Qt5OpenGL_DIR=C:\Qt\5.9.2\msvc2017_64\lib\cmake\Qt5OpenGL)
+- if "%APPVEYOR_BUILD_WORKER_IMAGE%"=="Visual Studio 2017" ( set Qt5Widgets_DIR=C:\Qt\5.9.2\msvc2017_64\lib\cmake\Qt5Widgets)
```

Comment:
I don't know the appveyor yaml syntax very well, but if there is a way do to multiline if statements that would make this way more readable.

Reply:
I tried various multi-lines and none of them work. Sorry :(